{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/cloudsim-legacy.json"}, "html": {"href": "#!/osrf/cloudsim-legacy"}, "avatar": {"href": "data/bytebucket.org/ravatar/{e799e7f6-0caa-464a-97f8-8605a37ebfb1}ts=python"}}, "type": "repository", "name": "cloudsim-legacy", "full_name": "osrf/cloudsim-legacy", "uuid": "{e799e7f6-0caa-464a-97f8-8605a37ebfb1}"}, "links": {"attachments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/140/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/cloudsim-legacy/issues/140.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/140/watch"}, "comments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/140/comments_page=1.json"}, "html": {"href": "#!/osrf/cloudsim-legacy/issues/140/task-failed-to-start-then-went-into"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/140/vote"}}, "reporter": {"display_name": "Brian Gerkey", "uuid": "{3d8b5763-9dad-4be7-85c2-a7ab6cd49173}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D"}, "html": {"href": "https://bitbucket.org/%7B3d8b5763-9dad-4be7-85c2-a7ab6cd49173%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:2e35028e-9ad8-4d6a-a3a5-275159a304a9/5fa9ec73-0fec-4860-a202-e34ae23b6a12/128"}}, "nickname": "Brian Gerkey", "type": "user", "account_id": "557058:2e35028e-9ad8-4d6a-a3a5-275159a304a9"}, "title": "Task failed to start then went into starting, then running", "component": null, "votes": 0, "watches": 0, "content": {"raw": "On a machine without the models installed, starting a task timed out at 30s, as expected, because the model download didn't complete in time.  We briefly saw the \"Task failed to start\" message, but the light never went red.  It blinked yellow, then eventually blinked blue.", "markup": "markdown", "html": "<p>On a machine without the models installed, starting a task timed out at 30s, as expected, because the model download didn't complete in time.  We briefly saw the \"Task failed to start\" message, but the light never went red.  It blinked yellow, then eventually blinked blue.</p>", "type": "rendered"}, "assignee": null, "state": "resolved", "version": null, "edited_on": null, "created_on": "2013-06-08T23:51:07.249347+00:00", "milestone": null, "updated_on": "2013-06-10T22:42:33.631444+00:00", "type": "issue", "id": 140}